src_assets/common/assets/web/configs/tabs/audiovideo/DisplayOutputSelector.vue

```diff
@@ -21,7 +21,12 @@ const outputNamePlaceholder = (props.platform === 'windows') ? '\\\\.\\DISPLAY1'
       {{ $tp('config.output_name_desc') }}<br>
       <PlatformLayout :platform="platform">
         <template #windows>
-          <pre>tools\dxgi-info.exe</pre>
+          <b>&nbsp;&nbsp;{</b><br>
+          <b>&nbsp;&nbsp;&nbsp;&nbsp;"device_id": "{de9bb7e2-186e-505b-9e93-f48793333810}"</b><br>
+          <b>&nbsp;&nbsp;&nbsp;&nbsp;"display_name": "\\\\.\\DISPLAY1"</b><br>
+          <b>&nbsp;&nbsp;&nbsp;&nbsp;"friendly_name": "ROG PG279Q"</b><br>
+          <b>&nbsp;&nbsp;&nbsp;&nbsp;...</b><br>
+          <b>&nbsp;&nbsp;}</b>
```

Comment:
I think we should use the `<pre>` tag like the other examples

Reply:
Done, but I'm still keeping `&nbsp;` for indentation. I could use `white-space: pre-wrap;`, but it uglyfies HTML a lot...